backend/consensus/base.py

```diff
@@ -352,10 +390,32 @@ def execute_transfer(
                 transaction.to_address, to_balance + transaction.value
             )
 
-        transactions_processor.update_transaction_status(
-            transaction.hash, TransactionStatus.FINALIZED
+        self.dispatch_transaction_status_update(
+            transactions_processor, transaction.hash, TransactionStatus.FINALIZED
         )
 
+    def dispatch_transaction_status_update(
+        self,
+        transactions_processor: TransactionsProcessor,
+        transaction_hash: str,
+        new_status: TransactionStatus,
+    ):
```

Comment:
nit: I know this was done to have DRY code, but I feel that creating a function just to log this might be an anti pattern. Maybe it makes more sense to have this log inside `transactions_processor.update_transaction_status`. WDYT?

Reply:
I felt the same way when doing it, I think you're right for the anti pattern. Originally I was doing it in the processor but @cristiam86 mentioned this should be in consensus. So we have anti pattern VS separation of concern.
We have two other options:
- Do like you said and like I did before @AgustinRamiroDiaz and move it to the transactions_processor.update_transaction_status method (no repeated logic, just a little bit of weird concern separation since we log a consensus event from the processor) if that's OK with @cristiam86 
- Repeat the log event at every place in the code where we update a tx status (repeated logic, but no anti pattern and keep concern in consensus)

Reply:
Let's leave it like this for now since
- we don't have a clear agreement
- it's a nit
- we need to go fast for the hackathon

Thanks for taking this into consideration :heart: